src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,4 +4,6 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] arg) {
+    }
 }
```

Comment:
Let’s remove this class and add a regular Main class with testing logic.

Reply:
can't add code here - getting <80% coverage doing so